src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentEditor.js

```diff
@@ -37,7 +37,7 @@ export class DocumentEditor extends Component {
     const request = _get(this.props, 'location.state', null);
     if (!request) return null;
     return {
-      documentRequestPid: request.pid,
+      documentRequestPid: request.id,
```

Comment:
`documentRequestPid: request.metadata.pid`, as above, to be verified if it works